benchmarks/entropy_adiabat/plugins/entropy_model.cc

```diff
@@ -24,12 +24,64 @@
 #include <aspect/utilities.h>
 
 #include <deal.II/base/table.h>
-
+#include <fstream>
+#include <iostream>
+#include <aspect/material_model/rheology/visco_plastic.h>
 
 namespace aspect
 {
   namespace MaterialModel
   {
+    namespace internal
```

Comment:
If you use the same class as the steinberger material model you can delete all of this copy (as long as you `#include <aspect/material_model/steinberger.h` at the top).